### ISSUE TITLE
Possibility to enable/disable Kubernetes tab on My Account section

### ISSUE THREAD

**Issue by ghost** [InseeFrLab/onyxia#467]
Hi, 

In some context, or by choice, a direct access to Kubernetes Server API is not allowed (ex: Kubernetes behind a Reverse Proxy).

On current version, `Kubernetes`  tab is always displayed even if it's not possible. So, when you select this tab, there is error and a white screen. 

Please, could you add a flag somewhere (ie, on region settings for instance) that give the possibility to enable or disable this tab ?

Regards,

**Comment by fcomte** [comment]
I think there is no need for a new flag. If no ` K8sPublicEndpoint ` is specify we should disable this kubernetes tab.

**Comment by garronej** [comment]
If its not already the case it's a bug.  
@cds-bercyhub  can you confirm you do not have a `K8sPublicEndpoint ` ?

**Comment by fcomte** [comment]
I confirm on my test env 
We see the kubernetes tab and when go there :

`TypeError: URL constructor: undefined is not a valid URL.`

This should be fix

**Comment by ddecrulle** [comment]
Should be solved with PR #471.

Bug was caused because region.kubernetes is not undefined when K8sPublicEndpoint is not specified. 

In this case, region.kubernetes is equals to : 

```ts
region.kubernetes = {url: undefined,keycloakParams: undefined}
```

**Comment by ghost** [comment]
After new release deployment including the PR, it seems to be OK fixed. Thanks :+1:
